### ISSUE TITLE
deploy failed after finish the compile task due to missing current directory

### ISSUE THREAD

**Issue by micheldenegri** [davidalger/capistrano-magento2#5]
Hello,

First of all thank you for this great gem, I wanted to share my experience so far. I had some problem after successfully running the `setup:di:compile-multi-tenant` task, it seams like it was expecting to find a current symlink. This is the error i got:

```
 DEBUG [5e3b1234]       Magento\AdminNotification\Controller\Adminhtml\Notification\MassRemove\Interceptor
 DEBUG [5e3b1234]       Magento\AdminNotification\Controller\Adminhtml\Notification\MarkAsRead\Interceptor
 DEBUG [5e3b1234]       Magento\AdminNotification\Controller\Adminhtml\Notification\MassMarkAsRead\Interceptor
 DEBUG [5e3b1234]   On *nix systems, verify the Magento application has permissions to modify files created by the compiler in the "var" directory. For instance, if you run the Magento application using Apache, the owner of the files in the "var" directory should be the Apache user (example command: "chown -R www-data:www-data <MAGENTO_ROOT>/var" where MAGENTO_ROOT is the Magento root directory).
  INFO [5e3b1234] Finished in 54.483 seconds with exit status 0 (successful).
  INFO [3889191b] Running /usr/bin/env rm -f var/di/relations.ser as user@example.com
 DEBUG [3889191b] Command: cd /home/user/example.com/releases/20160507060723 && /usr/bin/env rm -f var/di/relations.ser
  INFO [3889191b] Finished in 0.321 seconds with exit status 0 (successful).
 DEBUG [fd1c5a48] Running if test ! -d /home/user/example.com/current; then echo "Directory does not exist '/home/user/example.com/current'" 1>&2; false; fi as user@example.com
 DEBUG [fd1c5a48] Command: if test ! -d /home/user/example.com/current; then echo "Directory does not exist '/home/user/example.com/current'" 1>&2; false; fi
 DEBUG [fd1c5a48]   Directory does not exist '/home/user/example.com/current'
```

After i manually created a current symlink, i was able to finish the deployment on the next run.

cheers,

**Comment by yuklia** [comment]
how did you get such DEBUG messages?

**Comment by davidalger** [comment]
@micheldenegri Sorry about the long wait… I think the issue notification from GitHub got lost amidst the other Magento repository notifications. :) That said, this was fixed in commit c8793bef. If you update your gem to v0.2.1 or later the issue should no longer occur. Feel free to re-open if you run into the issue after updating your gem.

**Comment by micheldenegri** [comment]
No worries @davidalger, I was monitoring your repo and noticed you fixed the issue. I forgot to close the ticket when i saw that, so after i updated my gem i didn't encounter this problem anymore. 
cheers!
